src/object.c

```diff
@@ -1263,6 +1263,11 @@ struct redisMemOverhead *getMemoryOverheadData(void) {
         mem_total+=mem;
 
         mh->num_dbs++;
+
+        mh->db_overhead_hashtable_lut += kvstoreOverheadHashtableLut(db->keys);
```

Comment:
I suggest directly multiplying by sizeof(dictEntry *) to calculate the memory here.

Reply:
Or do the calculation inside `kvstoreOverheadHashtableLut` and `kvstoreOverheadHashtableRehashing`? Seems more reasonable that these two functions returns memory value.